.env

```diff
@@ -9,6 +9,15 @@ DATABASE_DB=airbyte
 # translate manually DATABASE_URL=jdbc:postgresql://${DATABASE_HOST}:${DATABASE_PORT/${DATABASE_DB}
 DATABASE_URL=jdbc:postgresql://db:5432/airbyte
 
+# By default, the Config Database is the same as the Job Database.
```

Comment:
I don't like the increased optionality around a custom config database here. Is there a reason we want to encourage separating this?

For users that were previously on the job database for configs, we don't want to handle migrating configs to the config database or having multiple versions of configs sitting around if we can help it.

It isn't clear to me what the benefit is, and it seems to just increase operational complexity.

Reply:
We do want to have the option to separate the two databases. But I don't think we want to encourage it, at least for self-deployed instances.

The benefits of separating the two databases are:
1. The jobs database and the configs database have different usage patterns. The former can grow much quicker than the latter (because we track job attempts there). 
2. The configs database contain secrets. So people may want extra security procedure around it.

What about if I remove the comments in the `.env` file, and only mention it in the documentation? In this way, we are not encouraging people to do it.

Reply:
Removing them from the `.env` will result in warning messages about missing environment variables. So I added them back with comments:

```bash
# Airbyte Internal Config Database, default to reuse the Job Database
# Usually you do not need to change them
CONFIG_DATABASE_USER=${DATABASE_USER}
CONFIG_DATABASE_PASSWORD=${DATABASE_PASSWORD}
CONFIG_DATABASE_URL=${DATABASE_URL}
```